docs/standard/library-guidance/dependencies.md

```diff
@@ -0,0 +1,95 @@
+﻿---
+title: Dependencies
+description: Best practice recommendations for managing NuGet dependencies in .NET libraries.
+author: jamesnk
+ms.author: James.NewtonKing
+ms.date: 09/20/2018
+---
+# Dependencies
+
+The primary way of adding dependencies to a .NET library is referencing NuGet packages. NuGet package references allow you to quickly reuse and leverage already written functionality, but they're a common source of friction for .NET developers. Correctly managing dependencies is important to prevent changes in other .NET libraries from breaking your .NET library, and vice versa!
+
+## Diamond dependencies
+
+It's a common situation for a .NET project to have multiple versions of a package in its dependency tree. For example, an app depends on two NuGet packages, each of which depends on different versions of the same package. A diamond dependency now exists in the app's dependency graph.
+
+![Diamond dependency](./media/diamond-dependency.png "Diamond dependency")
+
+At build time NuGet analyzes all the packages that a project depends on, including the dependencies of dependencies, and when multiple versions of a package are detected, rules are evaluated to pick one. Unifying packages is necessary because running side-by-side versions of an assembly in the same application is problematic in .NET.
+
+Most diamond dependencies are easily resolved, however they can create issues in certain circumstances:
```

Comment:
end the sentence at resolved with a period or semicolon and add missing comma after however